K2Bridge/JsonConverters/LeafAggregationConverter.cs

```diff
@@ -25,6 +25,9 @@ internal class LeafAggregationConverter : ReadOnlyJsonConverter
             var jo = JObject.Load(reader);
             var first = (JProperty)jo.First;
```

Comment:
We should probably not assume that PrimaryAggregation will be always on the first node. If "aggs" is the first node, its still a valid json and will break the code.

Reply:
I have finally confirmed that Kibana does send the primary aggregation as the first node; I had incorrectly used Jthe SON reformatted by MITMproxy, that had modified the order of elements in the JSON...

That said, the comment is still valid, we can't say for sure that the primary aggregation will always be the first element.